google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/MerchantCenterDynamicRemarketing.java

```diff
@@ -0,0 +1,389 @@
+
+import com.beust.jcommander.Parameter;
+import com.google.ads.googleads.examples.utils.ArgumentNames;
+import com.google.ads.googleads.examples.utils.CodeSampleParams;
+import com.google.ads.googleads.lib.GoogleAdsClient;
+import com.google.ads.googleads.v1.common.AdImageAsset;
+import com.google.ads.googleads.v1.common.AdTextAsset;
+import com.google.ads.googleads.v1.common.ImageAsset;
+import com.google.ads.googleads.v1.common.ManualCpc;
+import com.google.ads.googleads.v1.common.ResponsiveDisplayAdInfo;
+import com.google.ads.googleads.v1.common.UserListInfo;
+import com.google.ads.googleads.v1.enums.AdGroupStatusEnum.AdGroupStatus;
+import com.google.ads.googleads.v1.enums.AdvertisingChannelTypeEnum.AdvertisingChannelType;
+import com.google.ads.googleads.v1.enums.AssetTypeEnum.AssetType;
+import com.google.ads.googleads.v1.enums.CampaignStatusEnum.CampaignStatus;
+import com.google.ads.googleads.v1.errors.GoogleAdsError;
+import com.google.ads.googleads.v1.errors.GoogleAdsException;
+import com.google.ads.googleads.v1.resources.Ad;
+import com.google.ads.googleads.v1.resources.AdGroup;
+import com.google.ads.googleads.v1.resources.AdGroupAd;
+import com.google.ads.googleads.v1.resources.AdGroupCriterion;
+import com.google.ads.googleads.v1.resources.Asset;
+import com.google.ads.googleads.v1.resources.Campaign;
+import com.google.ads.googleads.v1.resources.Campaign.ShoppingSetting;
+import com.google.ads.googleads.v1.services.AdGroupAdOperation;
+import com.google.ads.googleads.v1.services.AdGroupAdServiceClient;
+import com.google.ads.googleads.v1.services.AdGroupCriterionOperation;
+import com.google.ads.googleads.v1.services.AdGroupCriterionServiceClient;
+import com.google.ads.googleads.v1.services.AdGroupOperation;
+import com.google.ads.googleads.v1.services.AdGroupServiceClient;
+import com.google.ads.googleads.v1.services.AssetOperation;
+import com.google.ads.googleads.v1.services.AssetServiceClient;
+import com.google.ads.googleads.v1.services.CampaignOperation;
+import com.google.ads.googleads.v1.services.CampaignServiceClient;
+import com.google.ads.googleads.v1.services.MutateAdGroupAdsResponse;
+import com.google.ads.googleads.v1.services.MutateAdGroupCriteriaResponse;
+import com.google.ads.googleads.v1.services.MutateAdGroupsResponse;
+import com.google.ads.googleads.v1.services.MutateAssetsResponse;
+import com.google.ads.googleads.v1.services.MutateCampaignsResponse;
+import com.google.ads.googleads.v1.utils.ResourceNames;
+import com.google.common.collect.ImmutableList;
+import com.google.common.io.ByteStreams;
+import com.google.protobuf.BoolValue;
+import com.google.protobuf.ByteString;
+import com.google.protobuf.BytesValue;
+import com.google.protobuf.Int32Value;
+import com.google.protobuf.Int64Value;
+import com.google.protobuf.StringValue;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.net.URL;
+
+public class MerchantCenterDynamicRemarketing {
```

Comment:
The name of this example is a bit non-standard. We usually use names of the form `<verb><type>`. Any particular reason from the different approach here?